openlibrary/plugins/openlibrary/js/service-worker-matchers.js

```diff
@@ -0,0 +1,42 @@
+/*
+// Contains all the matchers we use to decide wat to cache with the service workers.
+
+It is in a is a separate file to avoid this error when writing tests:
+    SyntaxError: Cannot use import statement outside a module
+    > 1 | import { ExpirationPlugin } from 'workbox-expiration';
+*/
+
+
+export function matchMiscFiles({ url }) {
+    const miscFiles = ['/static/favicon.ico', '/static/manifest.json', '/cdn/archive.org/analytics.js',
```

Comment:
favicon should generally be served from the root

```suggestion
    const miscFiles = ['/favicon.ico', '/static/manifest.json', '/cdn/archive.org/analytics.js',
```